android/assets/jsons/translations/template.properties

```diff
@@ -811,6 +811,8 @@ Your [ourUnit] captured an enemy [theirUnit]! =
 Your [ourUnit] plundered [amount] [Stat] from [theirUnit] = 
 We have captured a barbarian encampment and recovered [goldAmount] gold! = 
 An enemy [unitType] has joined us! = 
+After an unknown civilization entered the [eraName], we have recruited [spyName] as a spy! = 
+We have recruited [spyName] as a spy! = 
```

Comment:
Is it worth adding these strings to translation files so early? The espionage option is hidden so only devs would be able to access it but translators would have these strings appear in their files and might feel compelled to translate them (plus since this feature is so experimental these strings might change which makes constantly retranslating them annoying if that happens).

Reply:
The way I go about this, feature for feature, means that these strings should in theory not change anymore. Additionally, if I don't add them right now I feel like its likely I'll forget to add them later😅

Reply:
Yeah that works, looking at these strings made me realize that some translators would see them and might get confused.

Reply:
Why not add them in annotations? Or no poor translator can achieve 100% translation in a period of time any more [cry]